yarn.lock

```diff
@@ -12604,6 +12604,7 @@ __metadata:
     jest: ^27.3.1
     jest-fetch-mock: ^3.0.3
     jest-mock-process: ^1.4.1
+    jest-when: ^3.5.1
```

Comment:
TIL about `jest-when` 🙂

Reply:
It's pretty convenient. At first, I thought about writing inline abstractions, but then I saw that `jest-when` will be useful for more locations and more readable, so went with it.

Reply:
Yep, sounds quite useful for that purpose 👍